pcdsdevices/epics_motor.py

```diff
@@ -551,11 +571,9 @@ class SmarActOpenLoop(Device):
                          doc='Clear the current step count')
     set_metadata(step_clear_cmd, dict(variety='command-proc', value=1))
     # Scan move
-    scan_move_cmd = Cpt(EpicsSignal, ':SCAN_MOVE', kind='omitted',
```

Comment:
This will need at least an API change line in the release notes.

If we think there's anywhere this could be in use, we might consider some back-compat support.

Reply:
Good catch. There are very few places SmarAct stages are in use; MEC is the only other place I know of other than the laser system, and I don't expect that they are using this feature yet. @sfsyunusSLAC ?

Reply:
can confirm they are not